packages/dashboard/src/services/stub-authenticator.ts

```diff
@@ -10,10 +10,26 @@ export class StubAuthenticator
 
   readonly token?: string;
 
-  constructor(user = 'stub', token: string | undefined = undefined) {
+  constructor() {
     super();
-    this.user = user;
-    this.token = token;
+    this.user = 'stub';
+    // hardcoded token using the secret 'rmfisawesome', expires in 2035-01-01
+    // header:
+    // {
+    //   "alg": "HS256",
+    //   "typ": "JWT"
+    // }
+    // payload:
+    // {
+    //   "sub": "stub",
+    //   "preferred_username": "stub",
+    //   "iat": 1516239022,
+    //   "aud": "rmf_api_server",
+    //   "iss": "stub",
+    //   "exp": 2051222400
+    // }
+    this.token =
```

Comment:
this is awesome! Can you regenerate another token such that it is an admin?
You can verify it by seeing if Admin Actions are enabled for this user

Reply:
Added another token for user, also need to change the stub username to "admin". Without using other claims to determine if an user is admin, the only admin we can have is the default admin named "admin". We actually can use jwt claims to determine admin, but just not the `realm_access` claim as that is exclusive to keycloak, we need to define custom roles and document, then configure keycloak to sign those claims instead.